app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageFragment.kt

```diff
@@ -527,24 +524,12 @@ class NewMessageFragment : Fragment() {
         }
     }
 
-    override fun onStop() {
-        saveDraft()
-        super.onStop()
-    }
+    override fun onStop() = with(binding) {
 
-    private fun saveDraft() = with(newMessageViewModel) {
-        executeDraftActionWhenStopping(
-            action = if (shouldSendInsteadOfSave) DraftAction.SEND else DraftAction.SAVE,
-            isFinishing = requireActivity().isFinishing,
-            isTaskRoot = requireActivity().isTaskRoot,
-            subjectValue = binding.subjectTextField.text.toString(),
-            uiBodyValue = binding.bodyTextField.text.toString(),
-            startWorkerCallback = ::startWorker,
-        )
-    }
+        newMessageViewModel.subjectTextField = subjectTextField.text.toString()
+        newMessageViewModel.bodyTextField = bodyTextField.text.toString()
```

Comment:
Maybe we should add a comment to understand the context of these two lines of codes?